libs/designsystem/menu/src/menu.component.ts

```diff
@@ -96,31 +96,35 @@ export class MenuComponent implements AfterViewInit, AfterContentInit, OnDestroy
     ElementRef<HTMLElement>
   >;
 
+  @ContentChildren(ItemComponent) public kirbyItemComponents: QueryList<ItemComponent>;
+
   public floatingMenuIsShown: boolean = false;
   public FloatingOffset: typeof FloatingOffset = FloatingOffset;
   private scrollListenerDisposeFn: EventListenerDisposeFn;
   private focusedIndex = -1;
 
   @HostListener('keydown', ['$event'])
   _onKeydown(event: KeyboardEvent) {
+    if (this.kirbyItems.length === 0) {
+      console.warn('No items found within menu');
+      return;
+    }
     if (this.floatingMenuIsShown) {
       this.handleKeyDownForOpenedMenu(event);
     } else {
       this.handleKeyDownForClosedMenu(event);
     }
   }
 
-  private preventFurtherPropagation(event: KeyboardEvent) {
+  private preventDefaultAndStopImmediatePropagation(event: KeyboardEvent) {
     event.stopImmediatePropagation();
     event.preventDefault();
   }
 
   private getFirstInteractiveElement(el: HTMLIonItemElement) {
-    const controls = el.querySelectorAll<
-      HTMLIonToggleElement | HTMLIonRadioElement | HTMLIonCheckboxElement
-    >('ion-toggle:not([disabled]), ion-checkbox:not([disabled]), ion-radio:not([disabled])');
-
-    return controls[0];
+    return el.querySelectorAll<HTMLIonToggleElement | HTMLIonRadioElement | HTMLIonCheckboxElement>(
+      'ion-toggle:not([disabled]), ion-checkbox:not([disabled]), ion-radio:not([disabled])'
+    )[0];
```

Comment:
Consider simplifying:
```suggestion
    return el.querySelector<HTMLIonToggleElement | HTMLIonRadioElement | HTMLIonCheckboxElement>(
      'ion-toggle:not([disabled]), ion-checkbox:not([disabled]), ion-radio:not([disabled])'
    );
```